### PR TITLE
feat: Support prompt list format and add flashinfer-python hash

### PR DESCRIPTION
**Pull request by Zoe923**
This PR enhances the input and output capabilities for batch inference:
1. Update FlashInfer wheel package hash in requirements
2. Support `prompt` field as a list format (in addition to existing `prompt_batch`)

## Example Usage
```json
{
  "prompt": ["Query 1", "Query 2", "Query 3"],
  "generate_config": {
    "max_new_tokens": 100,
    "return_logits": true
  }
}

### CODE REVIEW

**Review by LLLLKKKK — CHANGES_REQUESTED** [review]

File: rtp_llm/frontend/frontend_worker.py [hunk 1]
```diff
@@ -233,10 +233,14 @@ def _format_response(
         loss = gen_responses.generate_outputs.generate_outputs[0].loss
         logits = gen_responses.generate_outputs.generate_outputs[0].logits
 
+        aux_info_dict = asdict(aux_info) if generate_config.aux_info else {}
+        if generate_config.return_logits and logits is not None:
+            aux_info_dict["logits"] = logits.tolist()
+
         response = PipelineResponse(
             response=generate_texts[0],
             finished=finished,
```
[the remark this answers is not in the document]
Comment (Zoe923) [reply rtp_llm/frontend/frontend_worker.py]: 好的
